docs/articles/networking/serialization.md

```diff
@@ -304,6 +304,43 @@ akka {
 }
 ```
 
+## Danger of polymorphic serializer
+One of the danger of polymorphic serializers is the danger of unsafe object type injection into 
+the serialization-deserialization chain. Hyperion attempted to alleviate this by blacklisting
```

Comment:
I'd add some caveats here:

1. This is an issue that primarily affects developers who allow third parties to pass messages directly to unsecured Akka.Remote endpoints, [a practice that we do not encourage](https://getakka.net/articles/remoting/security.html#akkaremote-with-virtual-private-networks) and do not see often in the wild.
2. This is an issue that also affects really, any other type of polymorphic serializer (i.e. JSON, BinaryFormatter, etc)
3. There are two approaches for solving this problem at point of the serializer: implement type filtering as shown here or implement schema-based serialization, which is more expensive to setup at first but fundamentally faster and more secure. Example: https://petabridge.com/cluster/lesson3

Reply:
Used some of your wordings to improve the documentation.